pymaker/shutdown.py

```diff
@@ -152,6 +152,12 @@ def cage(self, ilk: Ilk) -> Transact:
         assert isinstance(ilk, Ilk)
         return Transact(self, self.web3, self.abi, self.address, self._contract, 'cage(bytes32)', [ilk.toBytes()])
 
+    def snip(self, ilk: Ilk, clip_id: int) -> Transact:
```

Comment:
Question about things like `clip_id`, as I have seen this a few times.  It would be almost impossible for us to reach the max `int` in `clip_id`, yet `id` is still a `uint` on-chain.  Do we want to be more specific in cases where we get the signs wrong on these highly improbably cases?

Reply:
Python doesn't offer unsigned ints.